doc/source/reference/arrays.classes.rst

```diff
@@ -458,16 +467,22 @@ array actually get written to disk.
 
 Example:
 
->>> a = np.memmap('newfile.dat', dtype=float, mode='w+', shape=1000)
->>> a[10] = 10.0
->>> a[30] = 30.0
->>> del a
->>> b = np.fromfile('newfile.dat', dtype=float)
->>> print(b[10], b[30])
-10.0 30.0
->>> a = np.memmap('newfile.dat', dtype=float)
->>> print(a[10], a[30])
-10.0 30.0
+.. try_examples::
+
+  >>> import numpy as np
+
+  >>> a = np.memmap('newfile.dat', dtype=float, mode='w+', shape=1000)
+  >>> a[10] = 10.0
+  >>> a[30] = 30.0
+  >>> del a
+
+  >>> b = np.fromfile('newfile.dat', dtype=float)
+  >>> print(b[10], b[30])
+  10.0 30.0
+
+  >>> a = np.memmap('newfile.dat', dtype=float)
+  >>> print(a[10], a[30])
+  10.0 30.0
```

Comment:
I haven't tested whether every example works because there are way too many, but I did test this and this is one of the examples that doesn't, and fails with the error

```
ValueError: mmap length is greater than file size
```

and reducing the shape from `1000` to `100` doesn't help: `OSError: [Errno 43] No such device`

Reply:
I'm not sure if there is a way to enable the Pyodide file system or interact with Pyodide directly through a Pythonic interface like this.